classes/mail/variables/IssueEmailVariable.php

```diff
@@ -67,4 +71,33 @@ protected function getIssueUrl(): string
             $this->issue->getBestIssueId()
         );
     }
+
+    protected function getIssueToc(): string
+    {
+        $request = Application::get()->getRequest();
+
+        $submissions = Repo::submission()->getCollector()
+            ->filterByContextIds([$this->getContext()->getId()])
+            ->filterByIssueIds([$this->issue->getId()])
+            ->getMany();
+
+        if($submissions->count() <= 0) {
+            return '';
+        }
+
+        $list = $submissions->map(function ($submission) use ($request) {
```

Comment:
This will just include the submission titles, without the section information. I wonder whether it would be possible to generate this for the email by re-using `templates/frontend/objects/issue_toc.tpl` rather than trying to reimplement the functionality there over again...

Reply:
used the `templates/frontend/objects/issue_toc.tpl` but this is how it looks, not very good looking . 

<img width="1404" alt="CleanShot 2023-07-25 at 17 36 05@2x" src="https://github.com/pkp/ojs/assets/2587979/d0a66266-cc73-4086-a09c-bf69339ea385">

Reply:
I'm still on the fence about whether or not we should use the same template for the email and normal TOC presentations -- but in any case, it would be nice not to have to go through duplicating all the effort of fetching the data that would be included in the template. We're likely to badly maintain it and miss out on important features that way (thinking ordering, subscriptions, plugin-based interventions, etc).

We already have `IssueHandler::_setupIssueTemplate` made `static` so that it can be reused (see e.g. `pages/index/IndexHandler.php`, which uses it to present the current issue on the journal homepage when that option is selected). I can think of a few options...
- Add some flags to the existing `issue_toc.tpl` template to allow calling code to have better control over the presentation (e.g. showing/hiding the publication date, if we don't want that in the email)
- Reuse `IssueHandler::_setupIssueTemplate`, but implement a new template file that is a stripped-down version of the current `issue_toc.tpl`
- Re-use both `IssueHandler::setupIssueTemplate` and `issue_toc.tpl`, but experiment with styling the output of the current `issue_toc.tpl` in order to make it nicer when presented in an email

Reply:
Reused the `IssueHandler::_setupIssueTemplate` but not possible to make it prettier as this only load the HTML, not the CSS and to to do that we need to build a different template file . take a look at the current view and if we decide to build a new template file, we can do that 

<img width="1405" alt="CleanShot 2023-08-01 at 17 52 07@2x" src="https://github.com/pkp/ojs/assets/2587979/2007fea6-2d6e-414a-99f6-221ab19ee83b">